plugins/irma-console/irma-console.js

```diff
@@ -33,24 +36,26 @@ module.exports = (askRetry, askPairingCode) => {
     }
 
     _askPairingCode(askedBefore) {
-      // A new transition cannot be started within stateChange, so add call to javascript event loop.
-      Promise.resolve().then(() => {
+      return this._stateMachine.selectTransition(({validTransitions, isEndState}) => {
```

Comment:
So,  `selectTransaction` expects a callback which possibly has three parameters: `state`, `validTransitions` and `inEndState`. 

Here and below are some calls that look like `selectTransition(({validTransitions, isEndState}) => {...})`. 
I'm no expert on Javascript. Will they work correctly with this difference in naming (`inEndState` expected vs `isEndState` passed)? Or will `isEndState` become `undefined` in this callback?

Reply:
Oh right this will become `undefined` then. That is a mistake. I have to fix that. Sharp. If JavaScript would have typing these mistakes wouldn't be possible :(

Reply:
Fixed it.